docs/sources/auth/planning/index.md

```diff
@@ -122,13 +122,43 @@ Service account access tokens inherit permissions from the service account.
 
 ### 🚧 API keys
 
-## 🚧 How to work with roles?
+## How to work with roles?
 
-### 🚧 What are permissions?
+Roles are used to regulate which resources each user or service account can access within Grafana and what actions they're allowed to carry out.
 
-### 🚧 What are roles?
+Roles can be set manually through the UI or APIs, provisioned through Terraform or automatically synced through an external IAM provider.
 
-### 🚧 Grafana roles vs RBAC: Which one is for me?
+### What are roles?
+
+Grafana has a set of basic roles - organization administrator, editor and viewer - that determine user's permissions within an organization, such as access to edit data sources or create teams.
+Each user must have one of these roles assigned to them in each organization that they are a member of.
```

Comment:
not: I'd like to add links to other documentation on roles, permissions etc. However, there is a chance that we'll move them around as part of the doc refactor work. So I'm skipping the references now and will add them in later when the rest of the doc structure is clearer.

Reply:
I agree! I have run into these problems before.

What I'm doing is adding placeholders for doc references.

Example
```
APIs are used for [M2M] communication.
```

Once we start working on other sections, we can update this `[M2M]` reference with the proper link.